content/run.md

```diff
@@ -145,9 +145,14 @@ First, you need to change the global input data directory variable inside the sc
 
     dir_inputdata='/cluster/shared/noresm/inputdata/'
 
+```{Attention} You need to also choose what resolution global surface data you like to use? For example:
```

Comment:
The attention block format is not so pretty if you use a long line on the top. What about a small refrasing? 

```{Attention} Global surface data 
To change the resolution of the global surface data, e.g.  a 0.5x0.5-resolution add the file  `/cluster/shared/noresm/inputdata/lnd/clm2/surfdata_map/surfdata_360x720cru_16pfts_Irrig_CMIP6_simyr2000_c170824.nc` to the python script.
```

Reply:
Also, remember to check that the static page is properly built locally!

```
{Attention} 
```

is not a common `.md`-format, but comes from the template we are using.